onnxruntime/core/framework/tensorprotoutils.h

```diff
@@ -12,7 +12,7 @@
 #include "core/framework/ml_value.h"
 #include "core/framework/mem_buffer.h"
 #include "core/framework/tensor_external_data_info.h"
-#include "core/session/onnxruntime_c_api.h"
+#include "core/session/onnxruntime_cxx_api.h"
```

Comment:
>"core/session/onnxruntime_cxx_api.h" [](start = 9, length = 36)

This file has no code, possibly only types. Is there a way to get by with forward declarations?

Reply:
I'm not sure what you mean, to create a separate version of the C++ API header that just does forward declarations?